cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -97,7 +97,7 @@ object EvaluatorCommand : CommandWithHelp() {
             return if (evaluator.checkSyntax(script)) 0 else 2
         }
 
-        val evaluatorRun = evaluator.run(script)
+        val evaluatorRun by lazy { evaluator.run(script) }
```

Comment:
Commit message nits:
- typo in "Delay" (?)
- I do not really get the body text, s/is/if/?

Reply:
Improved the commit message.

Reply:
Yet another nit: "*the* check"

Reply:
Done.